trk/src/Navigation/AppNavigator.js

```diff
@@ -65,6 +71,11 @@ function ProfileStack() {
         component={FeedbackForm}
         options={{ title: 'Feedback Form', headerBackTitle: 'Back', headerTitleAlign: 'center' }}
       />
+        <Stack.Screen
+        name="Definition"
+        component={GlossaryDefinition}
+        options={{ title: 'Definition', headerBackTitle: 'Back', headerTitleAlign: 'center' }}
+      />
     </Stack.Navigator>
   );
```

Comment:
import and add glossaryDefiniton screen in navigation to be available in home and profile (the two places where climbDetail exists)